### PR TITLE
more translations

### PR DESCRIPTION
**Pull request by zryuek**

### CODE REVIEW

File: addons/sourcemod/translations/zombieriot.phrases.icons.txt [hunk 1]
```diff
@@ -11,37 +11,24 @@
...
-	{
-		"en" 		"The Messenger has accumulated chaos."
-	}
-	"Chaos Kahmlstein Arrived"
-	{
-		"en" 		"Kahmlstein, the rebellious chaos inflicted leader will annihilate you."
-	}
-	"The Purge Arrived"
-	{
-		"en" 		"The Purge has accumulated chaos."
```
Comment (Batfoxkid) [inline addons/sourcemod/translations/zombieriot.phrases.icons.txt:44]: How come the translation removals?

Comment (zryuek) [reply addons/sourcemod/translations/zombieriot.phrases.icons.txt:44]: Ah, my bad. Might have accidentally deleted some lines.

Comment (zryuek) [reply addons/sourcemod/translations/zombieriot.phrases.icons.txt:44]: Oh wait I see, I was editing a seperate older file saved on my pc which did not have these translations yet; and I just copy pasted the contents assuming it was up-to-date. I'll fix this in a bit.

File: addons/sourcemod/translations/zombieriot.phrases.icons.txt [hunk 2]
```diff
@@ -127,99 +133,95 @@
...
+		"chi"	"Donnerkrieg和Schwertkrieg回来了。"
 	}
 	"Diversionistico Spawn"
 	{
 		"en"	"Diversionistico Spawn"
-		"pt"	"Diversionistico Apareceu."
-	}
-	"Snipers Appear"
-	{
-		"en"	"Snipers Appear."
```
Comment (Batfoxkid) [inline addons/sourcemod/translations/zombieriot.phrases.icons.txt:211]: Phrase is missing along with many "pt" translations were removed.

**Review by Batfoxkid — CHANGES_REQUESTED** [review]

**Review by Batfoxkid — APPROVED** [review]

**Pull request merged**